docs/hour-of-code/educators-2021.html

```diff
@@ -398,12 +398,93 @@ <h5 class="display-font">Assign in Teams</h5>
                     </div>
                 </div>
             </section>
-
+            <section class="hoc-section">
+                <h3 class="display-font title sunflower">
+                    Free Trainings and Live Lessons
```

Comment:
"Trainings" should be singular - and I would say "Educator Training and Student Lessons" as the header to distinguish between the offerings